### PR TITLE
Filters support for IOU

### LINKED ISSUE #1113: Filters support for IOU

### PR DESCRIPTION
**Pull request by noplay**
Fix #1113

### CODE REVIEW

**Comment by grossmj** [comment]
Looks like filters are not applied.

**Comment by noplay** [comment]
I tested with delay

**Comment by noplay** [comment]
I problably tested on the first port only, perhaps their is an issue on how I count the ports.

File: gns3server/compute/base_node.py [hunk 1]
```diff
@@ -597,25 +597,29 @@ def _ubridge_apply_filters(self, bridge_name, filters):
...
+        :returns: Iterator building a list of filter
+        """
         i = 0
         for (filter_type, values) in filters.items():
             if isinstance(values[0], str):
                 for line in values[0].split('\n'):
                     line = line.strip()
-                    cmd = "bridge add_packet_filter {bridge_name} {filter_name} {filter_type} {filter_value}".format(
-                        bridge_name=bridge_name,
+                    yield "{filter_name} {filter_type} {filter_value}".format(
```
Comment (grossmj) [inline gns3server/compute/base_node.py]: I don't understand that, are you doing a yield on a string?

Comment (noplay) [inline gns3server/compute/base_node.py]: Yes it's on iterator returning filters.

**Comment by noplay** [comment]
Tested again the filters seem to apply

**Comment by grossmj** [comment]
Looks ok now, maybe I have pinged the local IP address when I tested...

**Pull request merged**